span-normalizer/span-normalizer/src/main/java/org/hypertrace/core/spannormalizer/jaeger/SpanFilter.java

```diff
@@ -88,14 +117,23 @@ public SpanFilter(Config config) {
    * the span should be dropped, false otherwise.
    */
   public boolean shouldDropSpan(
-      JaegerSpanInternalModel.Span span, Map<String, JaegerSpanInternalModel.KeyValue> tags) {
+      JaegerSpanInternalModel.Span span,
+      Map<String, JaegerSpanInternalModel.KeyValue> tags,
+      Map<String, JaegerSpanInternalModel.KeyValue> processTags) {
     if (anyCriteriaMatch(tags, spanDropCriterion)) {
       if (DROPPED_SPANS_RATE_LIMITER.tryAcquire()) {
         LOG.info("Dropping span: [{}] with drop criterion: [{}]", span, spanDropCriterion);
       }
       return true;
     }
 
+    if (anySpanDropFiltersMatch(spanDropFilters, tags, processTags)) {
+      if (DROPPED_SPANS_RATE_LIMITER.tryAcquire()) {
+        LOG.info("Dropping span: [{}] with drop filters: [{}]", span, spanDropFilters);
```

Comment:
This will log too much. I think we should move this to debug. Even above it should move to debug.
I'm assuming once we migrate the configs to this new format, we'll get rid of the above.

Reply:
Done